### DISCUSSION TITLE
一直卡在 start HTTP server @ 0.0.0.0:5244

### DISCUSSION THREAD

**Discussion by Deep-Dark-Forest** [AlistGo/alist#6772] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/91b5e760-9268-4efd-b602-e63497664f3e)
防火墙已经允许 alist

**Comment by anwen-anyi** [comment]
没问题就是这样的  你可以在网页访问服务了  

如果还是卡了你可以在黑色命令窗口内进行回车键

**Reply by Deep-Dark-Forest** [reply]
![image](https://github.com/user-attachments/assets/37f28c62-8fa8-4011-aa70-5db99fc2b81f)

按过回车了，还是没用......

**Reply by anwen-anyi** [reply, marked as answer]
> ![image](https://github.com/user-attachments/assets/37f28c62-8fa8-4011-aa70-5db99fc2b81f)
> 
> 
> 
> 按过回车了，还是没用......

没有加端口号
- 127.0.0.1:5244

**Reply by Simonzhang8** [reply]
> ![image](https://private-user-images.githubusercontent.com/131975043/350739234-37f28c62-8fa8-4011-aa70-5db99fc2b81f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE1NDA3MjQsIm5iZiI6MTcyMTU0MDQyNCwicGF0aCI6Ii8xMzE5NzUwNDMvMzUwNzM5MjM0LTM3ZjI4YzYyLThmYTgtNDAxMS1hYTcwLTVkYjk5ZmMyYjgxZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyMVQwNTQwMjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NTU3NzExOTQyYzNjZThiZTAwZjI5ZWU5NGEwMjBiNDM5NzgxODEyOGQ0MmExNGJmYzQ3NDI4MjA4ODQ2ZDViJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.TYOR9iUrlUtQG8NxDIuKOloVlu7sAewznapFojZ1Lyc)
> 
> 按过回车了，还是没用......

端口5244
